modules/special-rules/pages/a-dying-race.adoc

```diff
@@ -0,0 +1,16 @@
+= A Dying Race
+
+Every Eldar casualty is a tragedy, one step closer to the race's extinction.
+
+---
+
+Apply these rules to detachments with the {a-dying-race} ability:
```

Comment:
```suggestion
Where your detachment has the {a-dying-race} ability, apply these rules:
```
I'm suggesting this formulation of the sentence to better support other changes below.  Although, if we only have one short bullet then it will probably be better to lose the bullet and combine this:
"Where your detachment has the {a-dying-race} ability and it is destroyed, remove D3 points from your army morale."

Reply:
Doh, it should have been the singular 'this rule' version. 🙄

I'm not happy about changing the whole sentence, because I'm trying to keep the ability construction as consistent as possible. Based on some of the existing abilities like Assault, Engineers etc, I've been working with this sentence construction, followed by one or more bullet points:

> Apply these rules/this rule to [subjects] with the {keyword-attribute} ability:

If you would like me to use a _different_ construction that's fine, but whatever construction is chosen should be used consistently. That's also why I've used the same construction even when there's only one effect - it's adding a level of visual consistency that helps comprehension and retention.

If you don't have an alternative standard construction,
```suggestion
Apply this rule to detachments with the {a-dying-race} ability:
```

Reply:
It's a Detachment ability, so yes it's going to be 'has the {x} ability'.

However, this is one of the Detachment abilities that could theoretically be written as an Army rule instead. I just thought it was better categorised as a Detachment ability due to it being resolved per-detachment.

Reply:
Suggestion added to commit batch.